pkg/getty/etc/profile

```diff
@@ -0,0 +1,13 @@
+export CHARSET=UTF-8
+export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+export PAGER=less
+export PS1='(getty ns):\w\$ '
+export PS1='\h:\w\$ '
```

Comment:
Seeing Justin's point from below (init case), should `PS1=` be more intelligent?